website/src/components/CreatePortal.tsx

```diff
@@ -0,0 +1,46 @@
+import { type FunctionComponent, useState } from "react";
+import { VeraxSdk } from "@verax-attestation-registry/verax-sdk";
+import { useAccount } from "wagmi";
+
+export type SDKDemoProps = {
+  veraxSdk: VeraxSdk;
+  getTxHash: (hash: `0x${string}`) => void;
```

Comment:
0x${string} can be replaced by Address type

Reply:
Done for the entire website!